frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -49,7 +49,6 @@ pub mod pallet {
         <<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
 
     #[pallet::pallet]
-    #[pallet::generate_store(pub(crate) trait Store)]
```

Comment:
What about this one?
https://github.com/paritytech/substrate/pull/13365

Reply:
Yeah I saw this one, but it was not merged so decided to skip.
Should I add include this also?

Reply:
Ok, I noticed it's applied in frontier, hence my question.
But this has been merged after the `v0.9.40` so I guess it can be ignored for now.